### PR TITLE
Add doc about sink.max-broadcast-buffer-size

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

File: presto-docs/src/main/sphinx/admin/properties-exchange.rst [hunk 1]
```diff
@@ -87,3 +87,14 @@ tasks. If the task output is hash partitioned, then the buffer is
...
 network has high latency, or if there are many nodes in the cluster.
+
+``sink.max-broadcast-buffer-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type** ``data size``
+* **Default value:** ``200MB``
+
+Broadcast output buffer size for task data that is waiting to be pulled by
+upstream tasks. Broadcast buffer is used to store and transfer build side
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/admin/properties-exchange.rst:98]: The broadcast buffer is used

Comment (mosabua) [reply presto-docs/src/main/sphinx/admin/properties-exchange.rst:98]: If the buffer is too small, it prevents scaling of join probe side tasks, when new nodes are added to the cluster.

**Review by mosabua — CHANGES_REQUESTED** [review]
Minor rewording suggested

**Review by dain — APPROVED** [review]
I agree with @mosabua's suggestions, but otherwise looks good

**Pull request merged**